operator/tests/controller/metrics/200-access-unprivileged.yaml

```diff
@@ -0,0 +1,41 @@
+# This pod definition should match its siblings, except for the name, serviceAccount and command.
+apiVersion: v1
+kind: Pod
+metadata:
+  name: operator-metrics-unprivileged
+  labels:
+    test: metrics-access
+spec:
+  restartPolicy: Never
+  containers:
+  - name: run
+    image: registry.access.redhat.com/ubi9/ubi-minimal:latest
+    args:
+    - bash
+    - "-c"
+    # TODO(ROX-22287): use $TEST_NAMESPACE from Makefile once templating is supported
+    - >-
+        operator_ns="stackrox-operator"
+        url="https://rhacs-operator-controller-manager-metrics-service.$operator_ns.svc.cluster.local:8443/metrics";
+        set -u;
+        token="$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)";
+        for attempt in $(seq 5); do
+          echo Attempt $attempt:;
+          curl --insecure --dump-header /tmp/headers -H "Authorization: Bearer $token" "$url" > /tmp/response;
+          echo response:;
+          head -n 1 /tmp/headers;
+          echo checking response headers:;
+          if grep -Ei "^HTTP/[0-9.]+ 403" /tmp/headers; then exit 0; fi;
```

Comment:
I'm not sure why this should fail, and the previous one (`200-access-privileged`) should pass, if both use the same url and auth token. Maybe add a comment explaining this.

Reply:
Because of a different service account. I'll add a comment.

Reply:
Right, I didn't make the connection, thanks!